p2p/monero-p2p/src/client/handshaker.rs

```diff
@@ -167,11 +248,13 @@ where
         permit,
     } = req;
 
+    // A list of protocol messages the peer has sent during the handshake for us to handle after the handshake.
+    // see: MAX_EAGER_PROTOCOL_MESSAGES
```

Comment:
Allows IDE jump-to-definition
```suggestion
    // see: [`MAX_EAGER_PROTOCOL_MESSAGES`]
```